src/type/introspection.js

```diff
@@ -467,9 +466,23 @@ export const introspectionTypes: $ReadOnlyArray<*> = [
 
 export function isIntrospectionType(type: ?GraphQLType): boolean %checks {
   return (
-    isNamedType(type) &&
-    introspectionTypes.some(
-      introspectionType => introspectionType.name === type.name,
-    )
+    // Cannot use function calls to %checks unless flow's
+    // experimental.const_params=true is set.
+
+    // isNamedType(type)
+    type !== null &&
```

Comment:
`!!type &&` prettiers to `Boolean(type) &&` which is a function call which makes flow fail without `experimental.const_params=true`. And `type &&` fails because it's not returning a boolean.

Reply:
We can probably Boolean wrap the whole expression then. But also it would be great to avoid the additional `instanceof` checks - we've managed to corral them into their individual predicate functions so far